### PR TITLE
Feat(Query): OSS Bucket Has Static Website for Alicloud Terraform

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: OSS Bucket Has Static Website for Alicloud Terraform

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Comment by fjsnogueira** [comment]
Logo
**Checkmarx AST – Scan Summary & Details** – [24ea8c56\-83ed\-4b3e\-b359\-a3be9255acd4](https://eu.ast.checkmarx.net/projects/0244d5f3-72f2-4c91-8775-654a746fd66f/scans?id=24ea8c56-83ed-4b3e-b359-a3be9255acd4&branch=query%2Fstatic_website_exists)
### No New Or Fixed Issues Found

File: assets/queries/terraform/alicloud/oss_bucket_has_static_website/metadata.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "id": "2b13c6ff-b87a-484d-86fd-21ef6e97d426",
+  "queryName": "OSS Bucket Has Static Website",
+  "severity": "HIGH",
+  "category": "Insecure Configurations",
+  "descriptionText": "Checks if any static websties are hosted on buckets",
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/oss_bucket_has_static_website/metadata.json:6]: We should improve the query description for this query and `kics\assets\queries\terraform\aws\s3_static_website_host_enabled\test\positive_expected_result.json`

File: assets/queries/terraform/alicloud/oss_bucket_has_static_website/query.rego [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    
+    common_lib.valid_key(resource, "website")
+
+	result := {
+    	"debug": sprintf("%s", [resource]),
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("alicloud_oss_bucket[%s].website", [name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "For the resource 'website' to not be used.",
+		"keyActualValue": "The resource 'website' is being used.",
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/oss_bucket_has_static_website/query.rego:17]:
```suggestion
		"keyExpectedValue": "'website' to not be used.",
		"keyActualValue": "'website' is being used.",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Pull request merged**